app/src/main/java/com/example/keniphhomemaintenance/maintenance_screen/MaintenanceViewModel.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.keniphhomemaintenance.maintenance_screen
+
+import androidx.compose.runtime.*
+import androidx.lifecycle.ViewModel
+import com.example.keniphhomemaintenance.R
+
+class MaintenanceViewModel: ViewModel() {
+
```

Comment:
Hard coded the three maintenance items below. It is just dummy data for now.

Reply:
dummy thicc data